src/components/FormAdapters/SearchFieldAdapter.js

```diff
@@ -19,18 +44,20 @@ const SearchFieldAdapter = ({
           : required
       }
       inputProps={{
-        ...input,
+        ...inputCopy,
         ...inputProps
       }}
       hasWarning={meta.data.warning}
       hasError={meta.touched && meta.error}
       description={meta.error || meta.data.warning}
-      textField={textField}
+      textField={cTextField}
       onChange={value =>
-        input.onChange({ inputValue: value[textField], fieldValue: value })
+        input.onChange({ inputValue: value[textField], ...input.value, calculated: false })
       }
-      onType={value => input.onChange({ inputValue: value })}
+      onType={value => input.onChange({ inputValue: value, ...input.value, calculated: false })}
```

Comment:
the spread operator should be at the beginning of object, or inputValue will be overwritten by old values